adapted-articles/J/JesusChristTeachingsof.xml

```diff
@@ -14,77 +14,77 @@
-<p>The Gospels emphasize the divine necessity of the death of Jesus. In addition to the fulfillment motive, the idea of necessity is strong in the first prediction by Jesus of his approaching death. In John’s Gospel Jesus speaks of his “hour” several times in the earlier stages of his ministry as “not yet,” but in the later stages as having arrived. There is a sense of definite movement toward a climax, the hour undoubtedly being the hour of the Passion (cf. <a href="?bref=John.17.1">Jn 17:1</a>). There is not room for any disillusionment here. Jesus knew that only through the hour of death could the Father be glorified. The climax was according to an orderly plan.</p>
-<p>Jesus evidently regarded his death as in some ways a sacrifice. The clearest indication of this is in the words of institution at the Last Supper. The cup is connected with the blood of the new covenant, which is said to be for the “remission of sins” (<a href="?bref=Matt.26.26-28">Mt 26:26-28</a>). No explanation is given of the way in which the coming death, signified by the broken bread and poured-out wine, would bring about forgiveness of sins. But the immediate realization by the early church that Christ died for our sins (cf. <a href="?bref=1Cor.15.3">1 Cor 15:3</a>) shows that the importance of what Jesus said had been clearly grasped. The new covenant idea is parallel to the old covenant, which according to <a href="?bref=Exod.24.1-18">Exodus 24</a> was sealed with sacrificial blood; there can be little doubt that Jesus had this in mind when he spoke the words about the new covenant. It was also akin to the ideas expressed in <a href="?bref=Jer.31.1-40">Jeremiah 31</a>, referring to a covenant written on the heart rather than graven in stone.</p>
-<p>Another aspect of the death of Christ seen especially in John’s Gospel is the sense of completion that went with it. In Jesus’ prayer in <a href="?bref=John.17.1-26">John 17</a>, as he faces the cross, he declares that he has finished the work that the Father had given him to do (<a href="?bref=John.17.4">Jn 17:4</a>). This is reinforced by the cry from the cross, “It is finished,” which only John records (<a href="?bref=John.19.30">19:30</a>). This sense of accomplishment gives an air of triumph to what might otherwise have been considered a disaster.</p>
+<p class="fl">The announcement of the kingdom must be linked with Jesus’s approach to his own death, which Jesus saw as an integral part of his mission. He gave a series of indications that, when taken together, supply us with the meaning Jesus attached to his death within the entire range of his mission.</p>
```

Comment:
Original:

> Some have maintained that he ended his life in disillusionment, but a brief survey of his teaching about his own destiny is sufficient to dispel such a theory.

I think we need some of this, as the reference is to a major theme of 20th Century NT studies. "Some scholars have claimed that Jesus considered his ministry to have been a failure, but this is not supported by even a quick reading of the gospels."

Reply:
How about this, then:
> The announcement of the kingdom must be linked with Jesus’s approach to his own death, which Jesus saw as an integral part of his mission. Some scholars have claimed that Jesus considered his ministry to have been a failure, but this is not supported by even a quick reading of the gospels. Moreover, Jesus gave a series of indications that, when taken together, supply us with the meaning Jesus attached to his death within the entire range of his mission.

Reply:
Perfect!